site/content/blog/2021-03-01-whats-new-in-svelte-march-2021.md

```diff
@@ -0,0 +1,105 @@
+---
+title: What's new in Svelte: March 2021
+description: Call for Svelte Summit Speakers + Better SSR, Web Component and non-HTML5 compilation support.
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Lots to cover this month with releases from across the Svelte ecosystem. Most importantly, Svelte Summit Spring 2021 has an [Open Call for Speakers](https://sessionize.com/svelte-summit-spring-2021). **The deadlinen is March 14** so if you have an idea for a talk, submit it now!
+
+Let's dive into the news 🐬
+
+## What's new in `sveltejs/svelte`
+* SSR store handling has been reworked to subscribe and unsubscribe as in DOM mode. SSR stores should work much more consistently now (**3.31.2**, see [custom stores](https://svelte.dev/examples#custom-stores) and [Server-side component API ](https://svelte.dev/docs#Server-side_component_API))
+* Multiple instances of the same action are now allowed on an element (**3.32.0**, [xample](https://svelte.dev/repl/01a14375951749dab9579cb6860eccde?version=3.32.0))
+* The new `foreign` namespace should make it easier for alternative compile targets (like Svelte Native and SvelteGUI) by disabling certain HTML5-specific behaviour and checks (**3.32.0**, [more info](https://github.com/sveltejs/svelte/pull/5652))
+* Support for inline comment sourcemaps in code from preprocessors (**3.32.0**)
+* Destructured defaults are now allowed to refer to other variables (**3.33.0**, [example](https://svelte.dev/repl/0ee7227e1b45465b9b47d7a5ae2d1252?version=3.33.0))
+* Custom elements will now call `onMount` functions when connecting and clean up when disconnecting (**3.33.0**, checkout [this PR](https://github.com/sveltejs/svelte/pull/4522) for an interesting conversation on how folks are using Svelte with Web Components)
+* A `cssHash` option has been added to the compiler options to control the classname used for CSS scoping (**3.34.0**, [docs](https://svelte.dev/docs#svelte_compile))
+* Continued improvement to Typescript definitions
+
+For a complete list of changes, including bug fixes and links to PRs, check out [the CHANGELOG](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md)
+
+
+## New from `sveltejs/language-tools`
+
+- TypeScript's control flow pattern now works (as of TS 4.2) as of VS Code Svelte extension version 104.4.4 and `svelte-check` version 1.1.36.
+- For language server clients that don't support `didChangeWatchedFiles`, a fallback file watcher will be used instead
+- svelte-check 1.1.33 / extension version 104.4.0 introduced new highlighting rules for reactive statements, store accessors and element directives (likme `bind:` and `class:`)
```

Comment:
The reactive assignment highlighting was reverted for now because it introduced some problems.

```suggestion
- svelte-check 1.1.33 / extension version 104.4.0 introduced new highlighting rules for store accessors and element directives (likme `bind:` and `class:`)
```